pgjdbc/src/main/java/org/postgresql/jdbc/PgResultSet.java

```diff
@@ -671,6 +672,8 @@ public int getConcurrency() throws SQLException {
 
   }
 
+  private static final LocalDate LOCAL_DATE_EPOCH = LocalDate.of(1970, 1, 1);
```

Comment:
There is a constant in later Java versions, but not Java 8

Reply:
This should be better as a `// TODO` comment in code :)

Reply:
+1

Reply:
done.